crane/search/base.py

```diff
@@ -59,9 +59,14 @@ def _filter_result(self, result):
         :rtype:     bool
         """
         try:
+            # check against repositories in V1 data dict
```

Comment:
i suggest to swap the v1 check with the v2 check first, because most of the repos are v2 nowadays